### ISSUE TITLE
No securityContext support in Chart

### ISSUE THREAD

**Issue by Demonsthere** [cockroachdb/helm-charts#67]
**Description:**
Hello, I have noticed, that the chart does not expose pod or container securityContext, which makes it impossible to run on a PSP enabled cluster. This due to the user in the container:

```
Error: container has runAsNonRoot and image will run as root
```

**Comment by pha91** [comment]
Hello, Any news on this? Actual the hardcoded securityContext definition isn't complete while init job-container is still running priviledged :(

**Comment by prafull01** [comment]
Fixed as a part of #273
